catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/ShareableMetacardSharingPolicyPlugin.java

```diff
@@ -70,10 +71,10 @@
             .collect(Collectors.toSet()));
   }
 
-  private Map<String, Set<String>> getGroupPermission(WorkspaceMetacardImpl workspace) {
+  private Map<String, Set<String>> getGroupPermission(ShareableMetacardImpl shareableMetacard) {
     return ImmutableMap.of(
         Security.ACCESS_GROUPS,
-        Optional.of(workspace)
+        Optional.of(shareableMetacard)
             .map(w -> w.getAttribute(Security.ACCESS_GROUPS))
             .map(Attribute::getValues)
```

Comment:
✏️ Can now be replaced with: 
```
.map(s -> s.getAccessGroups())
```
which can be reduced down to a method reference.